ladybug/wea.py

```diff
@@ -772,6 +801,42 @@ def write(self, file_path, hoys=None, write_hours=False):
 
         return file_path
 
+    @staticmethod
+    def _get_datetimes(timestep, is_leap_year):
```

Comment:
I know this is a left over from the code and you didn't add it in this PR but now that I see it we can probably remove this and only use `AnalysisPreiod(timestep=timestep, is_leap_year=is_leap_year).datetimes`

Reply:
I wish it could be that simple but, unfortunately, we need a special formula here to deal with the fact that the EPW datetimes for radiation fall on the half-hour while, at finer timesteps, we expect the datetimes to start counting from 0:00 on Jan 1 and not 0:30. We could still use the AnalysisPeriod object to help us here but it's really only saving us a line of code and it comes at the expense of being slightly longer to run. So, I think it is best to leave it as it is for now.